src/app/app-design-tool/store/design-tool.reducers.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.
+//
+// Licensed under the Apache License, Version 2.0 (the 'License');
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as fromDesingToolActions from './design-tool.actions';
+import {debug} from '../../logger';
+import * as types from '../../app-interactive-map/types';
+import * as typesDesign from '../types';
+
+export interface DesignToolState {
+  allSpecies: types.Species[];
+  selectedSpecies: types.Species;
+  models: types.DeCaF.Model[];
+  selectedModel: types.DeCaF.Model;
+  products: typesDesign.Product[];
+  designStarted: boolean;
```

Comment:
Can you tell about this flag? How do you exactly use this?

Reply:
I am using this to show the floating menu and hide the big form :)